### PR TITLE
added support for zhimi.humidifier.cb2

### PR DESCRIPTION
**Pull request by sannoob**

### CODE REVIEW

File: miio/airhumidifier.py [hunk 1]
```diff
@@ -180,6 +183,11 @@ def motor_speed(self) -> Optional[int]:
     @property
     def depth(self) -> Optional[int]:
         """The remaining amount of water in percent."""
+
+        # MODEL_HUMIDIFIER_CB2 127 without water tank. 125 = 100% water
+        if self.device_info.model in [MODEL_HUMIDIFIER_CB2]:
```
Comment (zero734kr) [inline miio/airhumidifier.py:188]: Can you change this to
```py
if self.device_info.model == MODEL_HUMIDIFIER_CB2:
```
?

**Review by zero734kr — APPROVED** [review]

File: miio/airhumidifier.py [hunk 2]
```diff
@@ -180,6 +183,11 @@ def motor_speed(self) -> Optional[int]:
     @property
     def depth(self) -> Optional[int]:
         """The remaining amount of water in percent."""
+
+        # MODEL_HUMIDIFIER_CB2 127 without water tank. 125 = 100% water
+        if self.device_info.model == MODEL_HUMIDIFIER_CB2:
```
Comment (syssi) [inline miio/airhumidifier.py:188]: You could introduce a new boolean property to indicate the water tank is detached.

**Review by syssi — APPROVED** [review]
LGTM!

File: miio/airhumidifier.py (hunk 2 above)
Comment (rytilahti) [reply miio/airhumidifier.py:188]: Hi @syssi - do you want to get this change already into this PR, or shall we merge this already for the next release?

**Pull request merged**